elements/lisk-api-client/package.json

```diff
@@ -38,8 +38,10 @@
 	"dependencies": {
 		"@liskhq/lisk-codec": "^0.1.0-alpha.0",
 		"@liskhq/lisk-cryptography": "^3.0.0-alpha.0",
+		"isomorphic-ws": "4.0.1",
 		"pm2-axon": "4.0.0",
-		"pm2-axon-rpc": "0.6.0"
+		"pm2-axon-rpc": "0.6.0",
+		"ws": "7.4.0"
```

Comment:
`ws` seems to be used only in test, so `devDependencies`?

Reply:
The `lisk-api-client` can be used in Node environment as well so it do require `ws` to in dependencies. The documentation of `isomorphic-ws` also suggest to install both packages. 

https://github.com/heineiuo/isomorphic-ws#usage

Reply:
ah ok make sense!